### PR TITLE
Platform monitoring setup, internal vs external monitoring

### PR DESCRIPTION
**Pull request by sanderson**

### CODE REVIEW

File: content/platform/monitoring/_index.md [hunk 1]
```diff
@@ -16,8 +16,12 @@ another OSS or Enterprise TICK stack. They cover different potential monitoring
...
 and address anomalies as they happen.
 
-<!-- ## [Monitoring setup (Monitoring TICK with TICK)](#) -->
+## [Internal vs external monitoring](/platform/monitoring/internal-vs-external)
+An explanation of internal and external monitoring strategies for your Enterprise
+or OSS TICK stack with the pros and cons of each.
 
-<!-- ## [Internal monitoring vs external monitoring](#) -->
+## [Set up an external monitor](/platform/monitoring/external-monitor-setup)
+How to setup and external InfluxData TICK stack that monitors another Enterprise or OSS TICK stack.
```
Comment (noramullen1) [inline content/platform/monitoring/_index.md]:
```suggestion
How to set up and external InfluxData TICK stack that monitors another Enterprise or OSS TICK stack.
```

Setup is one word as a noun but two as a verb.

File: content/platform/monitoring/internal-vs-external.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+External monitoring consists of a TICK stack monitored by another TICK stack.
+This usually takes the form of an Enterprise cluster being monitored by an OSS TICK stack.
+It consists of Telegraf agents installed on each node in your primary cluster
+reporting metrics for their respective hosts to a monitoring TICK stack installed
+on a separate server or cluster.
+
+---
+
+_See the [Setup an external monitor](/platform/monitoring/external-monitor-setup)
```
Comment (noramullen1) [inline content/platform/monitoring/internal-vs-external.md]:
```suggestion
_See the [Set up an external monitor](/platform/monitoring/external-monitor-setup)
```

File: content/platform/monitoring/internal-vs-external.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
+---
+title: Internal vs external monitoring
```
Comment (stevebang) [inline content/platform/monitoring/internal-vs-external.md]: "vs" should always be spelled out.

File: content/platform/monitoring/internal-vs-external.md (hunk 2 above)
Comment (stevebang) [inline content/platform/monitoring/internal-vs-external.md]: It's better to say: "For more information about setting up an external monitoring TICK stack, see [Set up an external monitor](/platform/monitoring/external-monitor-setup)."  No need to say "See the ... page".  Also, it's better to start with what they're looking for.  https://developers.google.com/style/cross-references

File: content/platform/monitoring/internal-vs-external.md (hunk 3 above)
Comment (stevebang) [reply content/platform/monitoring/internal-vs-external.md]: Fixed by me.

File: content/platform/monitoring/external-monitor-setup.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
+---
+title: Set up an external monitor
+description: How to setup and external InfluxData TICK stack that monitors another Enterprise or OSS TICK stack.
```
[the remark this answers is not in the document]
Comment (stevebang) [reply content/platform/monitoring/external-monitor-setup.md]: Fixed by me.

**Pull request merged**